src/actions/auth/loginAction.js

```diff
@@ -0,0 +1,34 @@
+import axios from 'axios';
+import { toast } from 'react-toastify';
+import { LoginConstants } from '../actionTypes';
+import history from '../../utils/history';
+import { toastSuccess, toastFailure } from '../../utils/toast';
```

Comment:
Instead of having two functions for a toast why not have one function and just pass a parameter to specify if it successful or failed e.g 

> const displayToast = ('Message','success') =>{
Logic for the toast
}